wasm/src/graphics/webgl.rs

```diff
@@ -0,0 +1,126 @@
+use super::webgl_bindings::Gl2;
+use super::GraphicsApi;
+use crate::error::ClientError;
+use rask_engine::{math::Mat3, resources::Texture};
+
+mod imports {
+    extern "C" {
+        pub fn get_canvas_size() -> u32;
+        pub fn set_canvas_size(w: u32, h: u32);
+    }
+}
+
+fn get_canvas_size() -> (u32, u32) {
+    let x = unsafe { imports::get_canvas_size() };
+    (x >> 16, x & 0xffff)
+}
```

Comment:
why do we need this function instead of reading the dimensions from the synchronization memory?

Reply:
I am not sure, if the size field in the sync memory is guaranteed to be initialized, when the function is called. If yes, then this function is indeed not needed. Maybe there is an fancier solution for this.

Reply:
I'll have to verify that. Wold be nice to save a redundant function call

Reply:
You could set the syncronisation memoy on first call of the function to ensure the dimensions are right